lib/parser.js

```diff
@@ -31,6 +34,8 @@ function Parser(options) {
   this._onError = this._options.errorHandler || console.error;
   this._fileCache = {};
   this.dynamicIncludeSrc = [];
+  Parser.baseUrlMap = this._options.baseUrlMap;
+  Parser.rootPath = this._options.rootPath;
```

Comment:
These should be passed to `includeFile` and `renderFile` as `config` to better trace its use.
See `includeFile` in markbind-cli's `Page.prototype.resolveDependency`: [Page.js#L126-L129](https://github.com/MarkBind/markbind-cli/blob/1ff54e85f239778aed8c3ba4567759cc8f3f3fca/lib/Page.js#L126-L129)

Reply:
@rachx You can remove this after commit 99e0ce2